.github/workflows/ci-code.yml

```
@@ -92,6 +92,7 @@ jobs:
 
     - name: Install aiida-core
       run: |
+        pip install numpy==1.19.4
```

Let's at least ensure that the version from the requirements files is installed and add a comment as to why this work-around is necessary:
```suggestion
        # Work-around issue caused by pymatgen's setup process, which will install the latest 
        # numpy version (including release candidates) regardless of our actual specification:
        pip install `grep 'numpy==' requirements/requirements-py-${{ matrix.python-version }}.txt`
```